kbase-extension/static/kbase/js/util/string.js

```diff
@@ -149,6 +149,31 @@ define([], () => {
         return false;
     }
 
+    /**
+     * This brings sanity to the output object name string in a few steps:
+     * 1. the name is treated as a path, and reduced down to only the filename
+     *   e.g. /path/to/file.txt -> file.txt
```

Comment:
presumably nothing is done to remove suffixes -- this would just turn into `reads_txt` and a predicted output file would be `reads_txt_<suffix>`

Reply:
Yup. We kinda talked about that at some point, but never decided on it.
Also, this particular function is just to sanitize any string and make it a valid workspace name - it doesn't care about files or file suffixes. Though that could be a flag, later?